app/models.py

```diff
@@ -16,6 +16,56 @@ class User(db.Model):
     id = db.Column(db.Integer, primary_key=True)
     username = db.Column(db.String(64), unique=True, index=True)
     role_id = db.Column(db.Integer, db.ForeignKey('roles.id'))
+    # borrows = db.relationship('Borrow', backref='user', lazy='dynamic')
 
     def __repr__(self):
-        return '<User %r>' % self.username
\ No newline at end of file
+        return '<User %r>' % self.username
+
+
+class Book(db.Model):
+    __tablename__ = 'books'
+    id = db.Column(db.Integer, primary_key=True)
+    nazwa = db.Column(db.String(64), nullable=False)
```

Comment:
Nie mieszaj polskiego z angielskim!!!!!
```suggestion
    name = db.Column(db.String(64), nullable=False)
```